test/framework/package.py

```diff
@@ -224,6 +224,13 @@ def test_package(self):
             print "The Package File"
             print read_file(pkgfile)
 
+        ec_desc = EasyConfig(
+            os.path.join(test_easyconfigs, 't', 'toy', 'toy-0.0-gompi-1.3.12-test-description.eb'),
+            validate=False)
```

Comment:
do a hard check on the `description` with the backtick in it, to make it clear why this is an important test case